apps/studio/components/interfaces/Auth/AuthProvidersFormValidation.tsx

```diff
@@ -1379,9 +1379,16 @@ const PROVIDER_SAML = {
         'You will need to use the [Supabase CLI](https://supabase.com/docs/guides/auth/sso/auth-sso-saml#managing-saml-20-connections) to set up SAML after enabling it',
       type: 'boolean',
     },
+    SAML_EXTERNAL_URL: {
+      title: 'Set a SAML metadata URL different from what is defined with the API External URL',
```

Comment:
this is quite a mouthful for an input field label - can we shorten it? and probably shift the rest into the description

Reply:
this okay?

![image](https://github.com/user-attachments/assets/5916cf16-509e-4060-a523-0d637350b077)